src/routes/users.route.ts

```diff
@@ -19,6 +19,7 @@ class UsersRoute implements Routes {
     this.router.post(`${this.path}`, validationMiddleware(CreateUserDto, 'body'), this.usersController.createUser);
```

Comment:
These routes need some kind of auth validation by middleware? I wonder if we should use `authMiddleware` here?